specs/README.md

```diff
@@ -27,7 +27,7 @@ Use `spectrum` to create the Postman 2.x collection from the OpenAPI 3 API Speci
 The following will install the `spectrum` executable in the `~/go/bin` directory.
 
 ```bash
-$ go get github.com/grokify/spectrum
+$ go install github.com/grokify/spectrum@v1.10.3
```

Comment:
I had some trouble to install spectrum. The version used have to be the one in `go.mod`.  Otherwise, the file generated is completely different.